src/style.css

```diff
@@ -0,0 +1,157 @@
+html {
+  font-family: "Avenir", sans-serif;
+  margin: 0 auto;
+  background-color: #ffff;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+@font-face {
+  font-family: "Avenir";
+  font-style: normal;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+.searchBar {
+  position: relative;
+  margin-top: 20px;
+  display: flex;
+  width: 100%;
+}
+/*
+.Button {
+  display: block;
+  position: absolute;
+  margin-left: 26px;
+  margin-top: 25px;
+  width: 19px;
+  height: 19px;
+} */
+
+.searchImputbig {
+  height: 70px;
+  width: 100%;
+  background: white;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  padding-left: 62px;
+  line-height: 22px;
+  background-size: 19px;
+  background-position: 26px 25px;
+  font-size: 16px;
+  font-style: inherit;
+}
+
+.searchImputsmall {
+  height: 42px;
+  width: 100%;
+  background: white;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  padding-left: 32px;
+  line-height: 11px;
+  background-size: 11px;
+  background-position: 13px 15px;
+  font-size: 14px;
+  font-style: inherit;
+}
+
+.searchImputbig:hover {
+  width: 100%;
+  background: white;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+  background-size: 19px;
+  background-position: 26px 25px;
+}
+
+.searchImputsmall:hover {
+  background: white;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  padding-left: 32px;
+  line-height: 11px;
+  background-size: 11px;
+  background-position: 13px 15px;
+}
+
+.searchImputsmall:focus {
+  height: 42px;
+  width: 100%;
+  background: linear-gradient(180deg, white 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  padding-left: 32px;
+  line-height: 11px;
+  background-size: 11px;
+  background-position: 13px 15px;
+
```

Comment:
no need to change all styles on focus or hover
on focus should be changed background, on hover -  border and border-radius

Reply:
but the problem still persists

Reply:
a lot of problems was solved from previous request)

Reply:
remove redunant styles for hove and focus